### PR TITLE
(WEBRTC-731) Readme file updated.

### PR DESCRIPTION
**Pull request by guillermobattistel**
[WEBRTC-731 - Update docs for Push Notifications / CallKit and PushKit implementation](https://telnyx.atlassian.net/browse/WEBRTC-731)
---

New documentation added for: 
- SIP Credentials
- Push notifications: portal setup
- Push notifications: app setup
- PushKit
- CallKit

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -255,13 +278,211 @@ func initTelnyxClient() {
...
+
+For more information about `CallKit` you can check the official [Apple docs](https://developer.apple.com/documentation/callkit]). 
+
+__*General Setup:*__
+1. Import CallKit:
+```Swift
+import CallKit
+```
+2. Initialize CallKit
+```
```
Comment (KalpeshTalkar) [inline README.md:452]: Hey @guillermobattistel I think the indentation is 2 spaces here and do we need to specify Swift language to code block notation. (` ```Swift`)
On the webpage it looks like the code is not properly formatted:
readme-codeblock

Comment (guillermobattistel) [reply README.md]: Done

File: README.md [hunk 2]
```diff
@@ -190,8 +202,16 @@ extension ViewController: TxClientDelegate {
...
     }
 
+    func onPushCall(call: Call) {
+       // If you have configured Push Notifications and app is in background or the Telnyx Client is disconnected
+       // this delegate method will be called after the push notification is received.
+       // Update the current call with the incoming call
+       self.currentCall = call 
+    }
+    
     // You can update your UI from here base on the call states.
```
Comment (Oliver-Zimmerman) [inline README.md:215]: *based

**Review by Oliver-Zimmerman — APPROVED** [review]
LGTM 👍  besides the one small typo. I eventook some of it for my ReadME

**Pull request merged**